openapi-codegen/core/src/main/scala/sttp/tapir/codegen/EndpointGenerator.scala

```diff
@@ -85,14 +85,13 @@ class EndpointGenerator {
           case _ => throw new NotImplementedError("Can't create non-simple params to input")
         }
       }
-      .mkString("\n")
 
-    val rqBody = requestBody.fold("") { b =>
+    val rqBody = requestBody.map{ b =>
       if (b.content.size != 1) throw new NotImplementedError("We can handle only one requestBody content!")
-      s"\n.in(${contentTypeMapper(b.content.head.contentType, b.content.head.schema, b.required)})"
+      s".in(${contentTypeMapper(b.content.head.contentType, b.content.head.schema, b.required)})"
     }
 
-    params + rqBody
+    (params ++ rqBody).mkString("\n")
```

Comment:
Reason for this refactor is that if the param list was empty, but the body was present, you got a blank line just above the body - which bugged me